client/src/pages/Places.tsx

```diff
@@ -15,128 +33,237 @@ const OptimizedMap = lazy(() => import("../components/map/OptimizedMap"));
 
 
 
-const places = [
+
+interface Place {
+  id: number;
+  image: string;
+  location: string;
+  distance: string;
+  date: string;
+  price: string;
+  rating: number;
+  description?: string;
+  lat?: number;
+  lng?: number;
+  lat?: number;
```

Comment:
Add it only once (lat?:, lng?:) and remove the duplicate values, it will case errors